extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/ZipkinReporterProvider.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.jaeger.runtime;
+
+import javax.enterprise.inject.Produces;
+import javax.inject.Singleton;
+import javax.interceptor.Interceptor;
+
+import io.quarkus.arc.AlternativePriority;
+import io.quarkus.arc.properties.UnlessBuildProperty;
+
+@Singleton
+public class ZipkinReporterProvider {
+    @Produces
+    @Singleton
+    @UnlessBuildProperty(name = "quarkus.jaeger.zipkin.compatibility-mode", stringValue = "false", enableIfMissing = false)
+    @AlternativePriority(Interceptor.Priority.APPLICATION + 100)
```

Comment:
Is the `@AlternativePriority` needed? Are we expecting users to overload this with their own version?

Reply:
We don't need it, i removed.